projects/aca-content/src/lib/store/effects/node.effects.ts

```diff
@@ -308,6 +308,12 @@ export class NodeEffects {
       this.actions$.pipe(
         ofType<ExpandInfoDrawerAction>(NodeActionTypes.ExpandInfoDrawer),
         map((action) => {
+          this.router.events
+            .pipe(
+              filter((event) => event instanceof NavigationEnd),
+              first()
```

Comment:
I think you don't have to use filter, in this case you could with something like `first(event => event instanceof NavigationEnd)` which will emit only first value that matches the predicate

Reply:
good point, done